deploy/helm/scf/assets/operations/instance_groups/database.yaml

```diff
@@ -122,6 +121,9 @@
       processes:
       - name: mariadb_ctrl
         executable: /bin/bash
+        limits:
```

Comment:
Any particular reason to move these? I'd prefer having the `args` right after the `executable` as they together make the full command.

Reply:
it's hard for me to read where the keys belong if they're after the long script
can't enforce something like this .. but I'll put them before `executable`

Reply:
I see. This works too.